stock_custom_exchange_rate_date/views/stock_picking_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock.picking.form.inherit.custom.rate.date</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//label[@for='scheduled_date']" position="before">
+                <field name="purchase_id" invisible="True" />
```

Comment:
It is no longer required to add invisible fields to be able to reference them in other field's attributes, see https://github.com/odoo/odoo/pull/137031

Reply:
Done